src/Uno.Toolkit.UI/Behaviors/InputExtensions.cs

```diff
@@ -109,6 +141,36 @@ internal static bool IsEnterCommandSupportedFor(DependencyObject host)
 			return host is TextBox || host is PasswordBox;
 		}
 
+		private static void OnReturnTypeChanged(DependencyObject sender, DependencyPropertyChangedEventArgs e)
+		{
+			if (sender is Control control && (sender is TextBox || sender is PasswordBox) && e.NewValue is ReturnType returnType)
```

Comment:
Should we reset the TextBox/PasswordBox to `Default` if `NewValue is not ReturnType`?